install-dependencies.sh

```diff
@@ -104,6 +104,8 @@ fedora_packages=(
     curl
     rust
     cargo
+    rustfmt
+    clippy
```

Comment:
Please split this patch into a separate pull request. Changes to install-dependencies.sh require extra handling and should come alone (the change to rust.md can remain in the patch)